Gems/Prefab/PrefabDependencyViewer/Code/Source/MainWindow.cpp

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Interface/Interface.h>
+#include <Core/Core.h>
+#include <MainWindow.h>
+#include <QBoxLayout>
+#include <QLabel>
+#include <GraphCanvas/GraphCanvasBus.h>
+#include <GraphCanvas/Components/Nodes/NodeTitleBus.h>
+#include <GraphCanvas/Components/Nodes/NodeBus.h>
+
+namespace PrefabDependencyViewer
+{
+    GraphCanvas::GraphCanvasTreeItem* PrefabDependencyViewerConfig::CreateNodePaletteRoot()
+    {
+        const GraphCanvas::EditorId& editorId = PREFAB_DEPENDENCY_VIEWER_EDITOR_ID;
+        GraphCanvas::NodePaletteTreeItem* rootItem = aznew GraphCanvas::NodePaletteTreeItem("Root", editorId);
+
+        return rootItem;
+    }
+
+    /** Returns a bare minimum configuration to configure GraphCanvas UI
+    for the purpose of visualizing the Prefab hierarchy. */
+    PrefabDependencyViewerConfig* GetDefaultConfig()
+    {
+        // Make sure no memory leak happens here
```

Comment:
Its unclear what this comment is referring to.

Reply:
Resolved